lib/version_utils.pm

```diff
@@ -213,10 +215,6 @@ sub is_microos {
     return 0 unless $distri && $distri =~ /microos/;
     return 1 unless $filter;
 
-    # SUSE MicroOS will have versions 5.X, 6.X., the rest is openSUSE
-    my $type = $version =~ /^[56]\./ ? 'suse' : 'opensuse';
-    return $filter eq $type if ($filter =~ /opensuse|^suse/);
-
     my $version_is_tw = ($version =~ /Tumbleweed/ || $version =~ /^Staging:/);
```

Comment:
currently this function will be used only for o3 , right ? do we have Staging in o3 ? maybe we can simplify it even more ?

Reply:
Maybe, but I wanted to leave it as it was originally. If it needs improvement, I would leave it for another PR.